webapp/src/containers/createIncident/index.tsx

```diff
@@ -78,7 +78,9 @@ const UpdateState = () => {
     // Hide the modal and reset the states
     const hideModal = useCallback(() => {
         dispatch(resetGlobalModalState());
-        resetFieldStates();
+        setTimeout(() => {
+            resetFieldStates();
+        }, 500);
```

Comment:
why this needed?

Reply:
this is done to avoid showing the modal after closing the result panel.